test/tests/php-fpm-hello-web/run.sh

```diff
@@ -8,19 +8,22 @@ image="$1"
 
 # Build a client image with cgi-fcgi for testing
 clientImage='librarytest/php-fpm-hello-web:fcgi'
-docker build -q -t "$clientImage" - > /dev/null <<'EOF'
+docker build -t "$clientImage" - > /dev/null <<'EOF'
 FROM debian:jessie
 
-RUN set -x; apt-get update && apt-get install -y libfcgi0ldbl && rm -rf /var/lib/apt/lists/*
+RUN set -x && apt-get update && apt-get install -y libfcgi0ldbl && rm -rf /var/lib/apt/lists/*
 
 ENTRYPOINT ["cgi-fcgi"]
 EOF
 
 # Create an instance of the container-under-test
-cname="php-fpm-container-$RANDOM-$RANDOM"
-cid="$(docker run -d -v "$dir/index.php":/var/www/html/index.php:ro --name "$cname" "$image")"
+cid="$(docker run -d -v "$dir/index.php":/var/www/html/index.php:ro "$image")"
```

Comment:
This is unrelated, but I realized we weren't even using `cname` anymore so I removed it. :+1: